eureca_ubem/city.py

```diff
@@ -448,7 +455,13 @@ def buildings_creation_from_geojson(self, json_path):
                     volume=footprint_area * n_floors * floor_height * bd_data["VolCoeff"],
                     number_of_units=n_units, # 77 average flor area of an appartment according to ISTAT
                 )
-
+                pv=PV_system(name=f"Bd {name} PV system",
```

Comment:
Is it possible to associate the PV to the building? Something like:

pv = PV_System ( ..... )

Bd.add_pv_system( pv )

Reply:
added a list attribute (with an initial value equal to [ ]) to the building class (PV_systems). The photovoltaic class will be appended to the list as soon as it is created. (there can be multiple PV_systems)